src/test/java/duke/command/DeleteCommandTest.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import org.junit.jupiter.api.Test;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+
+public class DeleteCommandTest {
+    private final DeleteCommand deleteCommand = new DeleteCommand();
+    private final TaskList taskList = new TaskList();
+    private final Storage storage = new Storage();
+
+    @Test
+    public void testIsExit() {
+        assertFalse(deleteCommand.isExit());
+    }
+
+    @Test
+    public void testExecute() {
```

Comment:
Perhaps using a more intuitive variable name for tests,in the following format: featureUnderTest_testScenario_expectedBehavior()